reference/docs-conceptual/discover-powershell.md

```diff
@@ -77,197 +74,183 @@ standardized list of verbs. To get a full list of all possible verbs, use the `G
-Add         a           Common         Adds a resource to a container, or attaches an item to ano…
+The cmdlet returns a long list of verbs. The **Description** provides context for what the verb is
+meant to do. Here's the first few rows of output:
+
+```Output
+Verb    AliasPrefix   Group     Description
+----    -----------   -----     -----------
+Add     a             Common    Adds a resource to a container, or attaches an item to another item
+Clear   cl            Common    Removes all the resources from a container but does not delete the container
+Close   cs            Common    Changes the state of a resource to make it inaccessible, unavailable, or unusab…
+Copy    cp            Common    Copies a resource to another name or to another container
+Enter   et            Common    Specifies an action that allows the user to move into a resource
+Exit    ex            Common    Sets the current environment or context to the most recently used context
+...
 ```
 
 ## Find commands with Get-Command
 
-The `Get-Command` cmdlet returns a list of all available commands that are installed on your system.
-That list you get back, is quite large though. To make finding commands easier you can limit the
-amount of information that comes back. You can filter the response using either parameters or by
-using helper cmdlets.
+The `Get-Command` cmdlet returns a list of all available commands installed on your system. The list
+you get back is quite large. You can limit the amount of information that comes back by filtering
+the response using parameters or helper cmdlets.
 
 ### Filter on name
 
-You can filter the output of `Get-Command`, using different parameters. Filtering this way, is about
-querying a specific property on the command. The idea is that you specify what property you want to
-filter against, and then you provide a string that you want to match against. You'll therefore get a
-comparison that looks like this:
+You can filter the output of `Get-Command` using different parameters. Filtering allows you to find
+commands that have certain properties. The **Name** parameter allows you to find a specific command
+by name.
 
 ```powershell
-Get-Command -Name '*Process'
+Get-Command -Name Get-Process
 ```
 
-At this point, the filtering is trying to do an exact matching against the provided string argument.
-If you want to have more flexibility, in the comparison, you can use a wildcard `*`, that does
-pattern matching. The following code would look for all commands, who's name ends with process:
+```Output
+CommandType     Name              Version    Source
+-----------     ----              -------    ------
+Cmdlet          Get-Process       7.0.0.0    Microsoft.PowerShell.Management
+```
 
-Above, the parameter the `-Name` is used to filter. Apart from `-Name`, you can also filter on
-`-ParameterName` and `-Type`, for example.
+What if you wanted to find all of the commands the work with processes. You can use a wildcard `*`
```

Comment:
Nit: In the context here, I'm not sure if `can` or `could` is correct? I think the construction should be either "What if you want...? You can..." or "What if you wanted ...? You could..."

```suggestion
What if you wanted to find all of the commands the work with processes? You could use a wildcard `*`
```

```suggestion
What if you want to find all of the commands the work with processes? You can use a wildcard `*`
```